src/styles/partials/components/_pageheader.scss

```diff
@@ -0,0 +1,46 @@
+.dg_page-header {
+	//background-image: url(/sebin/r/t/prototype-subpage-landing-header-bkg.jpg);
+	background-repeat: no-repeat;
+	background-size: cover;
+	background: $pale-blue;
+	color: $white;
+	padding: $padding-largest;
+	text-decoration: none;
+	p {
+		color: $white;
+		font-family: 'Crimson Text', serif;
+		font-size: 25px;
+		font-style: italic;
+		padding-top: 25px;
```

Comment:
this should come from the closest padding variable. you will need to work with @sdurphy to get that number.